koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -439,6 +439,17 @@ def _write_incremental_logs(self, build_id, logs_dir):
             raise ContainerError("Build log finished but build still has not "
                                  "finished: %s." % build_response.status)
 
+    def _read_user_warnings(self, logs_dir):
+        log_filename = os.path.join(logs_dir, "user_warnings.log")
+
+        if os.path.isfile(log_filename):
+            try:
+                with open(log_filename, 'r') as logfile:
+                    self._user_warnings = logfile.read().splitlines()
```

Comment:
So this was defined in init as opbject, now it's suddenly a list of strings.

Can you return value instead of storing in self._user_warnings, I don't see any benefots of having is as object attribute.

Also in that forked process, user warnings can be just local variable